grabbit/src/main/groovy/com/twcable/grabbit/GrabbitConfiguration.groovy

```diff
@@ -50,10 +50,15 @@ class GrabbitConfiguration {
     }
 
 
-    public static GrabbitConfiguration create(@Nonnull String configJson) {
-        log.debug "Input: ${configJson}"
+    public static GrabbitConfiguration create(@Nonnull String configString) {
+        log.debug "Input: ${configString}"
 
-        final configMap = new JsonSlurper().parseText(configJson) as Map<String, String>
+        //YAML Specification doesn't support TAB characters in the Config. Technically, if you only have yaml files, then
+        //SnakeYAML library will correctly throw errors if tab characters are used. However, putting this in place to
+        //make it backwards compatible with the existing JSON configs since there were no strict rules to NOT use TAB
+        //characters in the JSON configs
+        configString = configString.replaceAll("\\t", "    ")
```

Comment:
Should explicitly have a test with a JSON config with TAB in it, since the "invisible" tab character in there now will likely disappear at some point in the future when  an editor/IDE converts hidden hard-tabs to spaces.